common/src/Microsoft.Azure.IIoT.Agent.Framework/src/Agent/Default/WorkerSupervisor.cs

```diff
@@ -110,12 +110,12 @@ public class WorkerSupervisor : IWorkerSupervisor, IDisposable {
             while (true) {
                 var workers = _agentConfigProvider.Config?.MaxWorkers ?? kDefaultWorkers;
                 while (_instances.Count < workers) {
-                    _logger.Information("Creating new worker...");
+                    _logger.Verbose("Creating new worker...");
                     var worker = await CreateWorker();
                 }
 
                 foreach (var stoppedWorker in _instances.Keys.Where(s => s.Status == WorkerStatus.Stopped)) {
-                    _logger.Information("Starting worker '{workerId}'...", stoppedWorker.WorkerId);
```

Comment:
>'{ [](start = 57, length = 2)

I guess generally these quotes mean the parameter is a string. I don't exactly know if this is more than a convention, it is at least a convention. Do you have a reason to change this? you know the saying, "When in Rome, do as romans do" :)

Reply:
Quotes are usually used as a delimiter when there is no structured logging. It's also not used consistently. I can bring it back consistently if you insist :)

Reply:
totally agree about consistency ... I don't mind how we do it as long as is consistent :). As I said, I don't know why it was used like that, I was curious what is the reason to change it.